Makefile

```diff
@@ -0,0 +1,15 @@
+integration_tests:
```

Comment:
If we are not adding any tests, we should not add dead scripts or configs for tests.

Reply:
I added some basic thing as starter that can be expanded

Reply:
That basic thing does not do anything helpful.

Reply:
removed